packages/flutter/lib/src/rendering/error.dart

```diff
@@ -9,6 +9,7 @@ import 'object.dart';
 
 const double _kMaxWidth = 100000.0;
 const double _kMaxHeight = 100000.0;
+const String line = '\n──────────────────────\n\n';
```

Comment:
It's this long without trying to read from size so it doesn't line wrap on small phones like iPhone SE